### ISSUE TITLE
Tenplay: 'ErrorFault' object has no attribute '_getitem_' [run(49)]

### ISSUE THREAD

**Issue by NCopeman** [xbmc-catchuptv-au/repo#25]
This issue has been happening for a while, on several different devices, and even on a clean install of openelec 6.0. The relevant part of the kodi log is

```

[kodi.log.txt](https://github.com/xbmc-catchuptv-au/repo/files/69196/kodi.log.txt)
ERROR: [plugin.video.catchuptv.au.ten] Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.video.catchuptv.au.ten/resources/lib/networktenaddon/__init__.py", line 49, in run
                                                self.plugin.run()
                                              File "/storage/.kodi/addons/script.module.xbmcswift2/lib/xbmcswift2/plugin.py", line 332, in run
                                                items = self._dispatch(self.request.path)
                                              File "/storage/.kodi/addons/script.module.xbmcswift2/lib/xbmcswift2/plugin.py", line 306, in _dispatch
                                                listitems = view_func(**items)
                                              File "/storage/.kodi/addons/plugin.video.catchuptv.au.ten/resources/lib/networktenaddon/play.py", line 42, in play
                                                media = api.get_media_for_video(videoId)
                                              File "/storage/.kodi/addons/plugin.video.catchuptv.au.ten/resources/lib/networktenvideo/api.py", line 278, in get_media_for_video
                                                amfHelper = BrightCoveAMFHelper(PLAYER_KEY, videoId, PAGE_URL, AMF_SEED)
                                              File "/storage/.kodi/addons/plugin.video.catchuptv.au.ten/resources/lib/networktenvideo/amf.py", line 52, in __init__
                                                self.data = self.__GetBrightCoveData()
                                              File "/storage/.kodi/addons/plugin.video.catchuptv.au.ten/resources/lib/networktenvideo/amf.py", line 96, in __GetBrightCoveData
                                                return response['programmedContent']['videoPlayer']['mediaDTO']
                                            TypeError: 'ErrorFault' object has no attribute '__getitem__'
```

The issue seems to only appear using TenPlay, and none of the other plugins.

[kodi.log.txt](https://github.com/xbmc-catchuptv-au/repo/files/69202/kodi.log.txt)

**Comment by ShogunQld** [comment]
I have the exact same issue happening on ubuntu HTPC. It's a standard install of ubuntu with just mySQL and Kodi installed. I'm using TENplay plugin version 0.5.10.

Adding my comment in the hope if gives this issue a bump :)

**Comment by ShogunQld** [comment]
I've done a bit more digging, and managed to log some info out of the .kodi/addons/plugin.video.catchuptv.au.ten/resources/lib/networktenvideo/amf.py script.

Firstly, the commented out debug on line 95 doesn't work because self.logging is a boolean, not a logging stream.

The actual response contains the following error:
com.brightcove.templating.PlayerAccessDeniedException : Attempt to access inactive Player, ID:2371615070001

However after decoding the response it is just: 
ErrorFault level=error code=u'Server.Processing'

So the decode isn't actually picking up the correct error, and it needs better error handling to catch and report this error properly.  But the real issue seems to be that something has changed in the streaming from channel ten (which is far beyond my level of knowledge)

Looks like there is nearly a couple of hundred other reports for this error as well, so hopefully the developer gets a chance to look into it soon :)

**Comment by NCopeman** [comment]
Don't know about you, but it seems to be resolved for me due to tenplay being updated to 0.5.11 about 3 days ago.

**Comment by kaza007** [comment]
Same here. All good.
